tests/sweep_framework/sweeps/data_movement/view/view_pytorch2.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple
+
+import torch
+import random
+import ttnn
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from models.utility_functions import torch_random
+
+TIMEOUT = 10
+# seed for random
+random.seed(0)
+
+
+def extract_brackets_content(line):
+    # Function to extract the content inside brackets
+    brackets_content = []
+    open_brackets = 0
+    current_content = ""
+
+    for char in line:
+        if char == "[":
+            open_brackets += 1
+            if open_brackets > 0:
+                current_content = ""  # Reset content inside the brackets
+        elif char == "]":
+            if open_brackets > 0:
+                brackets_content.append(current_content.strip())
+            open_brackets -= 1
+        elif open_brackets > 0:
+            current_content += char
+
+    return brackets_content
+
+
+def parse_md_file_simple_no_regex(file_path):
+    view_specs = []
+    i = 0
+    with open(file_path, "r") as file:
+        for line in file:
+            # Extract all sets of content inside brackets
+            brackets_content = extract_brackets_content(line)
+
+            if len(brackets_content) >= 3:  # Ensure we have both shape and size
+                shape_str = brackets_content[0]  # First set of brackets for shape
+                size_str = brackets_content[2]  # Third set of brackets for size
+
+                # Convert the shape and size strings to lists of integers
+                if "s" in shape_str or "s" in size_str:
+                    continue
+                shape = list(map(int, shape_str.split(",")))
+                size = list(map(int, size_str.split(",")))
+
+                # Append the dictionary to the list
+                view_specs.append({"shape": shape, "size": size})
+            i += 1
+
+    return view_specs
+
+
+parameters = {
+    "nightly": {
+        "view_specs": parse_md_file_simple_no_regex("sweeps/data_movement/view/view_trace.md"),
```

Comment:
We'll have to have this file checked in for this to run right?

Reply:
Yeah can we copy and paste it in here? If you want to update it in the future when PT2.0 people change things around then you can do some follow-ups.

Reply:
Should be able to just drop it in that folder yes

Reply:
Yup, for now just copy paste md file in. I was talking with Kevin I believe about how we should move forward. JSON is most likely best if we can get it.

Reply:
Can you drop in the file then we can merge this one. I just don't want to merge without the file because it will break CI.

Reply:
Have to make a new pr because of some merge I messed up with, but that one will contain the md